docker/convert.go

```diff
@@ -36,6 +37,16 @@ func Filter(vs []string, f func(string) bool) []string {
 	return r
 }
 
+func toMap(vs []string) map[string]struct{} {
```

Comment:
It's way to generic to not exists somewhere else, just didn't look at it for now 👼